propolis/src/block/crucible.rs

```diff
@@ -31,27 +31,19 @@ pub struct CrucibleBackend {
 
 impl CrucibleBackend {
     pub fn create(
-        disp: &Dispatcher,
         gen: u64,
         request: VolumeConstructionRequest,
         read_only: bool,
     ) -> Result<Arc<Self>> {
-        CrucibleBackend::_create(disp, gen, request, read_only)
+        CrucibleBackend::_create(gen, request, read_only)
             .map_err(map_crucible_error_to_io)
     }
 
     fn _create(
-        disp: &Dispatcher,
         gen: u64,
         request: VolumeConstructionRequest,
         read_only: bool,
     ) -> anyhow::Result<Arc<Self>, crucible::CrucibleError> {
-        slog::info!(
-            disp.logger(),
```

Comment:
Should we provide some other way for the backend here to get a `logger`?

Reply:
It has access to one via `SyncCtx` and `AsyncCtx` objects during operation.  The calling code was doing logging itself, so it seemed unnecessary to expose one here.